build/helpers/moment.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var moment = require('moment');
+
+module.exports = function (context, options) {
+	var date = options ? context : new Date();
+	var options = options || context;
+	var format = options.hash.format || 'YYYY-MM-DD ';
```

Comment:
Yeah. I think we should use `moment` if we use `moment` :). There are plenty of things exposed by its API to work with without having to use bare strings or Date-parsed stuff. Sound OK? If we use `moment` directly, I think we'll be OK here. Without the `utc` conversion. I *think* that's where you're going but wanted to confirm.